packages/web-components/fast-components/test/.mocharc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
No comment on this file, but should these be in a folder `__test__` like we have in `src` to keep naming the same?